Unit-11/main.js

```diff
@@ -0,0 +1,94 @@
+// exercise-1-1
+const returnsTheFunction = (array = '', n) => {
+  const newArray = []
+  for (let i = 0; i < n; i++) {
+    newArray.push(array)
+  }
+  return newArray
+}
+// exercise-1-2
+const returnsTheFunction = (array = '', n) => {
+  const newArray = []
+  let i = 0
+  while(i < n) {
+    newArray.push(array)
+    i += 1
+  }
+  return newArray
+}
+// exercise-2
+const returnsTheFunction = (array = []) => {
+  const newArray = []
+  for (let i = array.length - 1; i >= 0; i--) {
+    newArray.push(array[i])
+  }
+  return newArray
+}
+// exercise-3
+const deleteElement = (array = []) => {
+  const trueValues = []
+  for(let i = 0; i < array.length; i++) {
+    if(array[i] !== undefined && array[i] !== null && array[i] !== false && array[i] !== "") {
+      trueValues.push(array[i])
+    }
+  }
+  return trueValues
+}
+// exercise-4
+const convertArray = (data = []) => {
+  const array = {}
+  data.forEach(item => {
+    array[item[0]] = item[1]
+  })
+  return array
+}
+// exercise-5
+const sortTheArray = (array = []) => {
+  for(let i = 0; i < array.length - 1; i++) {
+    for(let j = i + 1; j < array.length; j++) {
+      if(array[i] > array[j]) {
+        let temp = array[i]
+        array[i] = array[j]
+        array[j] = temp
+      }
+    }
+  }
+  return array
+}
+// exercise-6
+const checkObject = data => {
+  if (typeof data === 'object') 
+    return true
+  else 
+    return false

```

Comment:
Ví dụ logic đang viết là đúng
Thì hãy viết hư thế này:
```suggestion
  return typeof data === 'object'
```